src/titiler/application/titiler/application/settings.py

```diff
@@ -10,6 +10,7 @@ class ApiSettings(pydantic.BaseSettings):
     cors_origins: str = "*"
     cachecontrol: str = "public, max-age=3600"
     root_path: str = ""
+    path_prefix: str = ""
```

Comment:
I don't love all the nested typed option stuff in titiler. I'm grateful you (or Jeff?) figured this out to make it work. It also looks like Pydantic might make classes in Python a bit nicer. It also seems like good Python is mostly typed.

Reply:
Yeah, the stack is certainly not simple. I'm hoping this contribution in particular can die someday in favor of [a Mangum setting](https://github.com/developmentseed/titiler/pull/515#issuecomment-1332268220). That should reduce the burden of updating this fork quite a bit smaller.

> It also looks like Pydantic might make classes in Python a bit nicer. It also seems like good Python is mostly typed.

I'm also excited to see all the typing in this ecosystem. I've largely [cargo culted](https://en.wikipedia.org/wiki/Cargo_cult_programming) changes to this point, so I can't say I even understand it all that well.